### PR TITLE
PT-3013: Pedigree editor form improvements

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

File: components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js [hunk 1]
```diff
@@ -1354,17 +1354,21 @@ define([
 
                 var suggestContainer = container.down('div.patient-newlink-container');
                 var suggestInput     = container.down('input[type=text].suggest-patients');
+                suggestInput.placeholder = " type patient name or identifier";
```
Comment (marta-) [inline components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js]: Why does it start with a space and small `t`?

Comment (veronikaslc) [inline components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js]: imho looks better that way

Comment (marta-) [inline components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js]: @veronikaslc If a space looks better for the placeholder, then it probably also looks better when there's a value entered, in which case the effect should be obtained via styling (padding left & right) instead.

I don't have a strong opinion regarding small vs capital first letter, as link as it's consistent across placeholders on the same form.

Comment (veronikaslc) [inline components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js]: I see, it's actually looks well without that space, so I'll just remove it
Regarding placeholders, in patient form we have it with both capital and small first letters, I personally prefer small, but it's super minor.

**Comment by sashaandjic** [comment]
+1

**Pull request merged**